tests/apollo/util/eliot_logging.py

```diff
@@ -16,18 +15,24 @@ def set_file_destination():
         # Single Apollo module loaded, test name known.
         test_name = f"{tests_names.pop()}_{storage_type}"
 
-    # Create logs directory if not exist
-    if not os.path.isdir("logs"):
-        os.mkdir("logs")
+    logs_dir = '/tmp/apollo/'
```

Comment:
`/tmp/apollo` is not mounted to the docker image. So it won't be available for the host(GitHub Action runner).
In general, I'd prefer all the logs to be stored in the `build` directory:
* it is ignored by git
* it is a part of the project' directory and can be naturally searched for logs
* by writing to the build we do not pollute the system(when built "natively")
* tests do not rely on system permissions
* `build` is actually mounted to the docker

Reply:
Oded, please consider Dima's comment here, with respect to the failure we saw in GitHub Actions:
https://github.com/vmware/concord-bft/pull/900/checks?check_run_id=1181065805

Wondering why this only fails in one of the jobs, and not all of them though ...

Reply:
Sure.